src/schema/common.ts

```diff
@@ -0,0 +1,26 @@
+export const linkedEventsBaseParams = `
```

Comment:
This is not big deal but gql-gen is using term args so maybe this could be called `linkedEventsBaseArgs`